extension/src/experiments/commands/index.ts

```diff
@@ -17,6 +17,11 @@ export const getBranchExperimentCommand =
   (cwd: string, name: string, input: string) =>
     experiments.runCommand(AvailableCommands.EXP_BRANCH, cwd, name, input)
 
+export const getRenameExperimentCommand =
+  (experiments: WorkspaceExperiments) =>
+  (cwd: string, oldName: string, newName: string) =>
+    experiments.runCommand(AvailableCommands.EXP_RENAME, cwd, oldName, newName)
```

Comment:
[Q] What happens if `newName == oldName`? Does the command fail?

Reply:
No, the command still runs. It returns as if name was changed.

Reply:
To close https://github.com/iterative/vscode-dvc/issues/4740 I'd get the repository here and move all of the extension-related details from the old name to the new name e.g.
```
const repository = experiments.getRepository(dvcRoot)
return repository.updateRenamedExperimentDetails(dvcRoot, oldName, newName)
```
`repository.updateRenamedExperimentDetails` will need to access the `experimentsModel` and update it's starred, plotted, etc status.